### PR TITLE
Issaw final quickstart

### PR DESCRIPTION
**Pull request by iwooden**
* Removing SSH auth from main template - leaving linked templates in case users want to add it back in easily
* NSG revamp - NSG linked template no longer needed, public bootnodes blocked for both MN and TX nodes
* TX node storage generation moved into linked template
* Updated documentation - including details from PDF and smart contract guide

### CODE REVIEW

File: ethereum-consortium-blockchain-network/README.md [hunk 1]
```diff
@@ -40,4 +67,114 @@ You can get a high level overview of the topology you just deployed by reviewing
...
+
+Once again, when the transaction is mined and committed into a block, the account balances will be reflected accordingly.  Note, wallet 1’s balance is deducted a bit more than 15 Ether, since you had to pay a mining fee to process the transaction.
+
+![consortium network](images/azure11.png) ![consortium network](images/azure12.png)
+
+## Smart Contract Guide
+
+Once you have an Ethereum account set up with some Ether, you can try deploying and interacting with smart contracts on your private blockchain. For this, we'll be using the Browser Solidity project, which integrates with Metamask and allows you to write smart contracts directly in your browser. Navigate [here](https://ethereum.github.io/browser-solidity/), and in the upper left click "New File".
+
+The sample code for this exercise can be found in the "sample-contracts.sol" file under the "scripts" directory of the GitHub repository. Copy and paste this code into the editor, and on the right you should see the compiled bytecode of your contracts, as well as buttons to create the contract on the blockchain or attach to an existing contract:
```
Comment (sidkri) [inline ethereum-consortium-blockchain-network/README.md]: We can add a relative link to the sample-contracts.sol file to make it easy for readers

File: ethereum-consortium-blockchain-network/README.md [hunk 2]
```diff
@@ -5,10 +5,35 @@
...
+
 ## Getting Started
 To begin, you will need an Azure subscription that can support deploying several virtual machines and standard storage accounts.  By default, most subscription types will support a small deployment topology without needing to increase quota.
 
-Once you have specified all parameters, specify a resource group and region to which to deploy all resources.  We recommend using a new separate resource group for ease of management and deletion.  Finally, acknowledge legal terms and click to ‘Create.’  Depending on the number of VMs being provisioned, deployment time can vary from a few minutes to tens of minutes.
+Once you have a subscription, go to [Azure Quick Start Templates](https://azure.microsoft.com/en-us/documentation/templates/) and search for “Ethereum Consortium Blockchain Network”.
+
+![consortium network](images/azure1.png)
+
+Select the template that is returned in the search results to take you to the Deployment page and then the ‘Deploy to Azure’ button to take you to the Template deployment wizard in the Azure Portal.  Note, you will be prompted to sign into your account and Azure subscription in the process if you are not already logged in.
```
Comment (sidkri) [inline ethereum-consortium-blockchain-network/README.md]: The above statement does not apply to the readme as the reader is already at the destination. I would remove it. We can include this info at the bottom as an FYI on how to find templates easily.  The image azure1.png should also not be here.

Comment (iwooden) [inline ethereum-consortium-blockchain-network/README.md]: Good catch, removed that language.

File: ethereum-consortium-blockchain-network/README.md (hunk 1 above)
Comment (iwooden) [inline ethereum-consortium-blockchain-network/README.md]: You're right, that would be better. Added a relative link.

**Pull request merged**